### PR TITLE
0.26.3

### PR DESCRIPTION
**Pull request by marcoag**
Once this is approved, I'll merge it, tag it, and then bloom-release it into `jazzy`. @MichaelOrlov FYI

### CODE REVIEW

**Comment by marcoag** [comment]
FYI had to force push to fix some weird DCO error, maybe due to some extra space or non printable character in the comment...

File: rosbag2_compression/CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,29 @@
...
+  cache for message definitions which is not changes.
+  * Don't call virtual methods from destructors
+  * Cleanup 'rosbag2_directory_next' after the test run
+  * Protect Writer::open(..) and Writer::close() with mutex on upper level
+  - Rationale: To avoid race conditions if open(..) and close() could be
+  ever be called from different threads.
+  * Bugfix for WRITE_SPLIT callback not called for the last compressed file
+  * Bugfix for lost messages from cache when closing compression writer
+  * Use dedicated is_open\_ flag instead of relying on storage\_
+  ---------
```
Comment (MichaelOrlov) [inline rosbag2_compression/CHANGELOG.rst]: @marcoag I would suggest to remove details from commit messages. We usually don't include them. Only links to the original PR and short description.

**Review by MichaelOrlov — COMMENTED** [review]
@marcoag I would suggest to remove details from commit messages. We usually don't include them. Only links to the original PR and short description.

File: rosbag2_compression/CHANGELOG.rst (hunk 1 above)
Comment (marcoag) [reply rosbag2_compression/CHANGELOG.rst]: True, sorry forgot to do that. Removed them in https://github.com/ros2/rosbag2/pull/1656/commits/114413f7b63a5dd7ac4b9ea53584e7691867a4e2.

**Review by MichaelOrlov — APPROVED** [review]
LGTM

**Review by clalancette — APPROVED** [review]

**Comment by ahcorde** [comment]
@clalancette @MichaelOrlov should we run CI ? or we can merge it ?

**Comment by MichaelOrlov** [comment]
I would merge it as is. 
@clalancette is it ok to merge without running CI?

**Comment by marcoag** [comment]
Agree with @MichaelOrlov, I don't think we need CI on this one, but will wait for @clalancette to confirm.

**Comment by clalancette** [comment]
No, let's go ahead and just merge it in and release it.

**Pull request merged**